src/internal/coredb/projects.go

```diff
@@ -98,6 +109,9 @@ func ListProject(ctx context.Context, db *pachsql.DB) (*ProjectIterator, error)
 	return iter, nil
 }
 
+//func ListProjectInTransaction(ctx context.Context, tx *pachsql.Tx, option ...ListProjectOption) ([]*pfs.ProjectInfo, error) {
```

Comment:
Remove this or implement.

Reply:
I think we have the convention of suffixing things like this with `Tx`.  And while it's technically more expressive to take both a context, and a `*pachsql.Tx`, since they do different things.  We have tried to pick on or the other.  The hope is that by omitting contexts in transaction methods, we will discourage long running operations in a transaction.